src/test/java/org/folio/rest/impl/OrdersImplTest.java

```diff
@@ -285,9 +290,31 @@ public void testListedPrintMonographInOpenStatus() throws Exception {
     // Set status to Open
     reqData.setWorkflowStatus(CompositePurchaseOrder.WorkflowStatus.OPEN);
 
+	  //Setup - store current date, month, year before setting DateOrdered
+	  Calendar cal = Calendar.getInstance();
+	  cal.setTime(new Date());
+	  int monthBeforeSetting = cal.get(Calendar.MONTH);
+	  int yearBeforeSetting = cal.get(Calendar.YEAR);
+	  int dateBeforeSetting = cal.get(Calendar.DATE);
+	  
     final CompositePurchaseOrder resp = verifyPostResponse(COMPOSITE_ORDERS_PATH, JsonObject.mapFrom(reqData).toString(),
       EXIST_CONFIG_X_OKAPI_TENANT_LIMIT_10, APPLICATION_JSON, 201).as(CompositePurchaseOrder.class);
 
+    //Test - verify date, month, year after getting DateOrdered
+    Date getDateOrdered = resp.getDateOrdered();
```

Comment:
Probably it is easier to use `java.time.LocalDate` i.e. something like
```java
LocalDate now = LocalDate.now();
...
LocalDate dateOrdered = resp.getDateOrdered().toInstant().atZone(ZoneId.of(ZoneOffset.UTC.getId())).toLocalDate();
...
assertThat(dateOrdered, equalTo(now));
```

Reply:
Yes, I have updated the code with this. @piotr-kalashuk